_posts/2020-10-28-push-it-to-the-limit.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Push It To The Limit
+categories:
+- blog
+tags:
+- MUD
+- Open Source
+- Optimization
+- Firebase
+- Lich
+---
+
+I've been hosting portions of the Lich framework I help maintain on [Firebase](https://firebase.google.com/) using a free tier plan for several years now. I became inured to the monthly e-mails reminding me that I was at 90% of my data limit showing up on the ~~30th~~ ~~29th~~ ~~28th~~... shit.
```

Comment:
This is a good place for a link to https://lichproject.org/

Or, and I prefer this, reword to provide a bit more context to the non-DR folks and link to something like https://www.rpherbig.com/2016/09/30/a-lot-can-happen-in-a-year.html

Reply:
Or, do the reword and link to "a lot can happen in a year", then link to https://rcuhljr.github.io/blog/2016/10/07/side-project-lessons-part1.html when you start talking about Lich itself.